### ISSUE TITLE
Cannot use span's for link targets, as they are not keyboard accessible

### ISSUE THREAD

**Issue by esteinborn** [ny/drupal-nys-global-nav#64]
unless we set the tabindex attribute on the spans, they are not keyboard accessible.

see: https://github.com/ny/global-navigation/issues/50

**Comment by esteinborn** [comment]
Solution built into special_menu_items: https://www.drupal.org/project/special_menu_items

## Compatibilities with other modules
If you use module than require to output a link, go in /admin/config/system/special_menu_items and use <a> as the HTML tag. You can even use <a href="#"> if you need to.

**Comment by itsgreggmarshall** [comment]
It's in the code and documentation, let's close this.
